pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -227,19 +228,11 @@ func (c *Controller) rejectRequest(pktIn *ofctrl.PacketIn) error {
 		isIPv6 = true
 	}
 
-	// Get the OpenFlow ports.
-	// 1. If we found the Interface of the src, it means the server is on this node.
-	// 	  We set `in_port` to the OF port of the Interface we found to simulate the reject
-	// 	  response from the server.
-	// 2. If we didn't find the Interface of the src, it means the server is outside
-	//    this node. We set `in_port` to the OF port of `antrea-gw0` to simulate the reject
-	//    response from external.
-	// 3. We don't need to set the output port. The pipeline will take care of it.
 	sIface, srcFound := c.ifaceStore.GetInterfaceByIP(srcIP)
-	inPort := uint32(config.HostGatewayOFPort)
-	if srcFound {
-		inPort = uint32(sIface.OFPort)
-	}
+	dIface, dstFound := c.ifaceStore.GetInterfaceByIP(dstIP)
+	isSvcTraffic := c.isServiceTraffic(pktIn, c.antreaProxyEnabled, dstMAC, dstFound)
```

Comment:
you don't need to pass `c.antreaProxyEnabled` as a parameter if `isServiceTraffic` is already a method on the `Controller` instance

Reply:
Good one. Changed.